d2l-input-search.html

```diff
@@ -44,23 +43,10 @@
 				padding-left: 2.4rem;
 			}
 
-			input::placeholder {
-				@apply --d2l-input-placeholder;
-			}
-
-			input:disabled {
-				@apply --d2l-input-disabled;
-			}
-
-			input:hover:disabled {
-				@apply --d2l-input-disabled-hover;
-			}
-
 			input:hover,
 			input:focus,
 			.d2l-input-search-hover input,
 			.d2l-input-search-focus input {
-				@apply --d2l-input-hover;
```

Comment:
I think is going to break. We apply `d2l-input-search-hover` (and focus) to the parent container whenever the search/clear buttons get focus. This maintains the "hover" style on the input even when technically it doesn't have focus or hover.

Reply:
You might be able to fix it with by splitting it out so that `.d2l-input-search-hover input, .d2l-input-search-focus input { @apply --d2l-input-hover; }` is still there?

Reply:
Ah good call, I see how its supposed to work now.  I think something is broken though, i See it workign on Polymer 1 but not Polymer 2 in the old repo

Reply:
Cool, I can have a look as I'm in here and need to fix a bunch of things that aren't working with IE11 and Edge (surprised, surprise!).